bigtable/google/cloud/bigtable/table.py

```diff
@@ -199,6 +207,60 @@ def row(self, row_key, filter_=None, append=False):
         else:
             return DirectRow(row_key, self)
 
+    def append_row(self, row_key):
+        """Create a :class:`~google.cloud.bigtable.row.AppendRow` associated with this table.
+
+        For example:
+
+        .. literalinclude:: snippets_table.py
+            :start-after: [START bigtable_table_append_row]
+            :end-before: [END bigtable_table_append_row]
+
+        Args:
+            row_key (bytes): The key for the row being created.
+
+        Returns:
+            A row owned by this table.
+        """
+        return AppendRow(row_key, self)
+
+    def direct_row(self, row_key):
+        """Create a :class:`~google.cloud.bigtable.row.DirectRow` associated with this table.
+
+        For example:
+
+        .. literalinclude:: snippets_table.py
+            :start-after: [START bigtable_table_direct_row]
+            :end-before: [END bigtable_table_direct_row]
+
+        Args:
+            row_key (bytes): The key for the row being created.
+
+        Returns:
+            A row owned by this table.
+        """
+        return DirectRow(row_key, self)
+
+    def conditional_row(self, row_key, filter_):
+        """Create a :class:`~google.cloud.bigtable.row.ConditionalRow` associated with this table.
+
+        For example:
+
+        .. literalinclude:: snippets_table.py
+            :start-after: [START bigtable_table_conditional_row]
+            :end-before: [END bigtable_table_conditional_row]
+
+        Args:
+            row_key (bytes): The key for the row being created.
+
+            filter_ (:class:`.RowFilter`): (Optional) Filter to be used for
```

Comment:
Do we need to keep the `_` on `filter_`?

Reply:
Of course. `filter` is a [built-in function](https://docs.python.org/3/library/functions.html#filter), if we'll erase `_`, we'll override this function, that is completely sin. It's Python naming standard

Reply:
I thought it may be a reserved name at this point, I just wanted to make sure this wasn't a leftover from a previous iteration. Thanks @IlyaFaer